docs/source/api.rst

```diff
@@ -34,7 +34,7 @@ Supported File Types
 Category    File Types
 ========== ========================================================================================================
 Plaintext   ``.eml``, ``.html``, ``.json``, ``.md``, ``.msg``, ``.rst``, ``.rtf``, ``.txt``, ``.xml``
-Images      ``.png``, ``.jpg``, ``.jpeg``, ``.tiff``, ``.bmp``
+Images      ``.png``, ``.jpg``, ``.jpeg``, ``.tiff``, ``.bmp``, ``.heic``
```

Comment:
Totally unrelated thought - we should link to this table in the api readme rather than maintain a separate table of supported types.

Also, be sure to bump the [supported mimetypes](https://github.com/Unstructured-IO/unstructured-api/blob/d9413d52e3dfe75201498a8d81290b4ab2ce36d2/prepline_general/api/general.py#L65) in the api!